preprocessors/autour/autour.py

```diff
@@ -62,7 +62,7 @@ def get_map_data():
         return validated
 
     # Check if request is for a map
-    if 'graphic' in content:
+    if 'graphic' in content or 'highChartsData' in content:
```

Comment:
Check for coordinates/placeID instead of this. Better to avoid needing to constantly update when unrelated features are introduced and instead only update when relevant features are introduced (that you'd need to support here anyway).